test/clj/athenaeum/handlers/user_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns athenaeum.handlers.user-test
+  (:require [clojure.test :refer :all]
+            [athenaeum.fixtures :as fixtures]
+            [athenaeum.handlers.user :as user]))
+
+(use-fixtures :once fixtures/load-config fixtures/set-datasource)
+(use-fixtures :each fixtures/clear-tables)
+
+(deftest login-test
+  (testing "Returns status 200, adds id-token cookie to response on successful id token verification"
+    (let [req {:headers {:id-token "valid-id-token"}}
+          res (with-redefs [user/verify-id-token (constantly true)
+                            user/get-session-id (constantly "new-session-id")]
```

Comment:
Instead of mocking this here, connect to Redis in the test and verify that the session key is set as expected.